create-bin-data.c

```diff
@@ -80,7 +80,7 @@ int main(int argc, char *argv[])
 	char perm[9];
 	fgets(buf, sizeof(buf), fp);
 	numRead = sscanf(buf, "%d %s %d %d", &canonical, perm, &isGraphlet, &numEdges);
-	assert(numRead == 2 || numRead == 4);
+	// assert(numRead == 2 || numRead == 4); This causes an error in make
```

Comment:
If this causes an error then that's a major problem. The line *must* contain either 2, or 4, elements on it.